### PR TITLE
(PC-31874) feat(Offer): Create `See more` button on new XP Cine block

### PR DESCRIPTION
**Pull request by inesmouandjolobe**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-31874

## Flakiness

If I had to re-run tests in the CI due to flakiness, I add the incident [on Notion][2]

## Checklist

I have:

- [X] Made sure my feature is working on web.
- [X] Made sure my feature is working on mobile (depending on relevance : real or virtual devices)
- [X] Written **unit tests** native (and web when implementation is different) for my feature.
- [X] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change

## Screenshots

**delete** _if no UI change_

| iOS | Desktop - Chrome |
| :-----------: | :---: |
| Simulator Screenshot - iPhone 15 - 2024-10-07 at 13 21 13 | Capture d’écran 2024-10-07 à 13 23 59 |

### CODE REVIEW

File: src/ui/components/buttons/ButtonSecondary.ts [hunk 1]
```diff
@@ -8,14 +8,14 @@ import { Logo as InitialLoadingIndicator } from 'ui/svg/icons/Logo'
...
-  ({ icon, disabled, textSize, theme, ...rest }) => {
+  ({ icon, disabled, textSize, theme, color, ...rest }) => {
     let Icon
 
     if (icon) {
       Icon = styled(icon).attrs({
         color: disabled
           ? theme.buttons.disabled.secondary.iconColor
-          : theme.buttons.secondary.iconColor,
+          : color ?? theme.buttons.secondary.iconColor,
```
Comment (xlecunff-pass) [inline src/ui/components/buttons/ButtonSecondary.ts:18]: Une autre manière de l'écrire pour un peu plus de visibilité selon moi

```typescript
disabled || !color
          ? theme.buttons.disabled.secondary.iconColor
          : theme.buttons.secondary.iconColor
```

**Review by xlecunff-pass — APPROVED** [review]
PR qualitative

File: src/ui/components/buttons/ButtonSecondary.ts (hunk 1 above)
Comment (inesmouandjolobe) [reply src/ui/components/buttons/ButtonSecondary.ts:18]: J'ai l'impression que ça ne fait pas la même chose ?

Comment (inesmouandjolobe) [reply src/ui/components/buttons/ButtonSecondary.ts:18]: est ce qu'écrit ainsi te convient ?

`    const defaultColor = color ?? theme.buttons.secondary.iconColor
    if (icon) {
      Icon = styled(icon).attrs({
        color: disabled ? theme.buttons.disabled.secondary.iconColor : defaultColor,
        size: theme.buttons.secondary.iconSize,
      })``
    }`

**Pull request merged**